lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -1199,3 +1199,16 @@ function dosomething_campaign_node_validate($node, $form, &$form_state) {
     }
   }
 }
+
+  /**
+   * @param object $campaign
+   *   Loaded campaign node to check if given feature is on
```

Comment:
yeah I agree it's not actually the node. it's like the campaign variables of the node, which might be equivalent to what you said but I'm not positive. regardless, not the node.